### ISSUE TITLE
Website navigation issue

### ISSUE THREAD

**Issue by recri** [djipco/webmidi#236]
**Description**
The website is pinning the browser to the first section of each page, it fights scrolling off that section with scrollbar or scrollwheel, the first nav item on the right remains selected even when you select a later item.

**Environment:**
Chrome stable recently updated on latest released Ubuntu Linux also regularly updated.

**Details**
I go to "https://webmidijs.org/docs/", the page loads with Step 1 selected in the right nav menu.  I can scroll the page Title offscreen and no further.  I can appear to select Step 2 or one of the other right nav menu items, but the selection is immediately reverted to Step 1.   I can, with concentrated effort drag the scroll bar down the page, but I can't read anything as the manual position fighting with the automated position flickers the page position up and down.  Same thing happens on all the pages I've visited with right nav menues.

**Comment by djipco** [comment]
Dear Roger,

Thanks for reporting this. Unfortunately, I do not currently have a Linux box to test this. However, I did check it on both macOS and Windows and the problem you describe does not occur.

The website is built using [Docusaurus](https://docusaurus.io/). Would it be possible for you to check other websites created with Docusaurus to see if the problem occurs with them also?

They have a [showcase](https://docusaurus.io/showcase). For instance, these sites seem to also use the right navigation menu:

* https://docs.dyte.io/docs/home/introduction
* https://reactnative.dev/docs/getting-started
* https://docsearch.algolia.com/docs/what-is-docsearch
* https://jestjs.io/docs/getting-started

**Comment by recri** [comment]
Fascinating.

The first, second, and fourth scroll correctly and the right nav menu
tracks the section displayed.

The third shows the same problem as webmidijs.

>
>    - https://docs.dyte.io/docs/home/introduction
>    - https://reactnative.dev/docs/getting-started
>    - https://docsearch.algolia.com/docs/what-is-docsearch
>    - https://jestjs.io/docs/getting-started
>
> I also tried webmidijs with opera and firefox on my Linux laptop, they
both work.  It also works on AndroidChrome, and on the old install on a
Samsung Chromebook, and  I'll test the updated 91.0.4472.167 build in a
moment.  It works there, too.

Well, at least I have several workarounds, so I can get back to reading
your docs.

Best of luck sorting this out.  Let me know if you rebuild and want a test.

-- rec --

**Comment by djipco** [comment]
An [issue](https://github.com/facebook/docusaurus/issues/6669) has been filed with the Docusaurus team. I will wait for their feedback...

**Comment by djipco** [comment]
The Docusaurus team [replied](https://github.com/facebook/docusaurus/issues/6669#issuecomment-1038545360). Can you check if the problem is related to the "Experimental web platform features" option being turned on?

**Comment by recri** [comment]
Ah, sorry, didn't mean to ghost you, just lost sync with my inbox.

Yes, disabling "experimental web platform features" in linux chrome stable
on ubuntu fixed the problem on the webmidijs.org website.

Thank you for the solution.

Yesterday I discovered that a Teensy 4.0 can send enough NRPNs/second to
crash chrome and lock up the Linux input system.  Had to force power down
the laptop several times before I tracked down the culprit.  This wasn't
webmidijs but my own PWA under development.

Here I am reading more than 256k 3 byte midi messages/second from the Alsa
raw interface:
Fri Mar 04 22:10:02 EST 2022 786432 bytes read
Fri Mar 04 22:10:03 EST 2022 786432 bytes read
Fri Mar 04 22:10:03 EST 2022 786432 bytes read
Fri Mar 04 22:10:04 EST 2022 786432 bytes read
Fri Mar 04 22:10:05 EST 2022 786432 bytes read
Fri Mar 04 22:10:05 EST 2022 786432 bytes read
Fri Mar 04 22:10:06 EST 2022 786432 bytes read
Fri Mar 04 22:10:07 EST 2022 786432 bytes read
Fri Mar 04 22:10:07 EST 2022 786432 bytes read
Fri Mar 04 22:10:08 EST 2022 786432 bytes read
Fri Mar 04 22:10:09 EST 2022 786432 bytes read
That can happen if you define a midi message sent counter in your NRPNs,
and then send it whenever the count changes.

Cheers,

-- rec --
